src/Filter/IfThen.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+namespace Topolis\Bolt\Extension\ContentImport\Filter;
+
+use Exception;
+use Silex\Application;
+use Topolis\Bolt\Extension\ContentImport\Filter\Traits\ApplyFilter;
+use Topolis\Bolt\Extension\ContentImport\IFilter;
+use Topolis\FunctionLibrary\Collection;
+
+class IfThen implements IFilter {
+
+    use ApplyFilter;
+
+    public static function filter($input, $parameters, Application $app, $values, $source)
+    {
+        $output = $input;
+
+        foreach ($parameters as $param) {
+            $field = $param['field'] ?? false;
+
+            // Digs deeper in to the field or use the field value for the check
+            $value = Collection::get($output, $field, $output);
+            $state = true;
+
+            // Made like this to exten with the other Checks ne, gt, lt ...
```

Comment:
Using a fixed location for the condition type and a fixed location for it's evaluation parameter would make the code less convoluted (reduces need for all those issets). Also a switch/case would improve readability further.